modules/protocol/resources/META-INF/smithy/smithy4s.meta.smithy

```diff
@@ -163,3 +163,14 @@ structure generateOptics {}
 ///  via extending scala.util.control.NoStackTrace instead of Throwable.
 @trait(selector: "structure :is([trait|error])")
 structure noStackTrace {}
+
+/// Allows users to manually add imports to files of generated shapes.
+/// This would be helpful when some shape needs a specific import in order
+/// to compile. espically in the case you want to compose refinement types
+/// and other validators. `providerImport` should be an import that the
+/// target shape is required to compile.
+@trait(selector: "structure :not([trait|error])") // todo remove trait
+structure scalaImports {
```

Comment:
I'd make it a 

```smithy
list scalaImports {
  member: Import
} 
```

Reply:
fixed in https://github.com/disneystreaming/smithy4s/pull/1550/commits/627af24f1b96596d0b9a8c0fe30458974c43f392